src/libs/ReportUtils.js

```diff
@@ -79,11 +80,11 @@ Onyx.connect({
     callback: (val) => (doesDomainHaveApprovedAccountant = lodashGet(val, 'doesDomainHaveApprovedAccountant', false)),
 });
 
-let allPolicies;
+let allPolicies = {};
```

Comment:
Why are you making this change and also to `allReports`?

Reply:
I want to default the collection to an empty object so that I can attempt to access values under a given key without it throwing an error because allReports is undefined.

Reply:
OK, I am very hesitant to make that change because it could have unknown effects on other code referencing these collections. I would suggest handling the `undefined` object where you're trying to access it. You can see most of the code references `allReports` or `allPolicies` is using something like `_.find()` (which handles `undefined`), `lodashGet()`, or doing a falsy check before trying to access them. I think these are better methods so that nothing is assumed about the default state of the collections.